lib/bridge/MatrixHandler.js

```diff
@@ -674,6 +678,19 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
             return;
         }
     }
+    else if (cmd === "!bridgeversion") {
+        try {
+            const BridgeVersion = require("../../package.json").version;
+            const msgText = `BridgeVersion: ${BridgeVersion}`;
+            let notice = new MatrixAction("notice", msgText);
+            yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
+        }
+        catch (err) {
+            let notice = new MatrixAction("notice", `Failed to get bridgeversion: ${err.message}`);
```

Comment:
we probably don't want to send the error to the user, but do want to log it.

Reply:
is this place ok ? : req.log.info(....

Reply:
I think so, although it'd be better to use `error` instead of `info`.